code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -132,7 +133,7 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 
 	init()
 
-	if(!scrambledcodes && !camera)
```

Comment:
Won't no longer checking for scrambled codes change things? Is scrambled code still needed in other places?

Reply:
It is still required in other spots. It was used for syndicate borgs that spawned with this turned on.
It indirectly also meant they had no cameras. I gave them a FALSE on the `has_camera` instead.

Reply:
In that case, why not just give maint drones scrambled codes? From a quick search it seems its only other use is preventing the borg from being assigned to an AI, it would even let you safe a A.scrambledcodes || istype(A,/mob/living/silicon/robot/drone) in unsorted.dm

Reply:
It would also prevent them from being blown up.
scrambledcodes means they are rogue in most of the code where it is used. It was abused in the New to see if it should have a camera or not which in the case of a syndicate borg (only one who has it on newing) should not have one.